### PR TITLE
Adds trekchems to the Medibots.

### PR DESCRIPTION
**Pull request by Iamgoofball**
requested by literally everyone ever

### CODE REVIEW

**Comment by PKPenguin321** [comment]
i love you

**Comment by GunHog** [comment]
Acceptable :+1:

File: code/game/machinery/bots/medbot.dm [hunk 1]
```diff
@@ -46,10 +46,10 @@
...
 	desc = "International Medibot of mystery."
 	skin = "bezerk"
-	treatment_oxy = "omnizine"
-	treatment_brute = "omnizine"
-	treatment_fire = "omnizine"
-	treatment_tox = "omnizine"
+	treatment_oxy = "tricordrazine"
+	treatment_brute = "tricordrazine"
+	treatment_fire = "tricordrazine"
+	treatment_tox = "tricordrazine"
```
Comment (Cheridan) [inline code/game/machinery/bots/medbot.dm]: Isn't omnizine generally better, since medbots won't OD you?

Comment (PKPenguin321) [inline code/game/machinery/bots/medbot.dm]: agreed actually, i love trekchems as much as the next guy but you'll rarely see more than one mysterious medibot so they should never be ODing you

Comment (GunHog) [inline code/game/machinery/bots/medbot.dm]: Omnizine is PAINFULLY slow. While it heals lots of damage types, it takes forever to do, and Mysterious medibots should be more powerful and faster than normal ones.

Comment (Iamgoofball) [inline code/game/machinery/bots/medbot.dm]: yeah, phil nerfed the heal speed on it when he did 1tick

**Pull request merged**